library/ssl_client.c

```diff
@@ -883,6 +898,17 @@ int mbedtls_ssl_write_client_hello( mbedtls_ssl_context *ssl )
     else
 #endif /* MBEDTLS_SSL_PROTO_TLS1_2 && MBEDTLS_SSL_PROTO_DTLS */
     {
+
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3) && \
+    defined(MBEDTLS_KEY_EXCHANGE_SOME_PSK_ENABLED)
+        if( binders_len > 0 )
+        {
+            MBEDTLS_SSL_PROC_CHK(
+                mbedtls_ssl_tls13_write_pre_shared_key_ext_binders(
+                      ssl, buf + msg_len - binders_len, buf + msg_len ) );
+        }
```

Comment:
This is fix for `create_psk_binder` wrong. That is found in #6079 
```suggestion
        if( binders_len > 0 )
        {
            unsigned char binders = buf + msg_len - binders_len;
            mbedtls_ssl_add_hs_hdr_to_checksum( ssl,
                                                MBEDTLS_SSL_HS_CLIENT_HELLO,
                                                msg_len );

            ssl->handshake->update_checksum( ssl, buf, binders - buf );

            MBEDTLS_SSL_PROC_CHK(
                mbedtls_ssl_tls13_write_binders_of_offered_psks(
                      ssl, binders, buf + msg_len ) );

            ssl->handshake->update_checksum( ssl, binders, binders_len );
        }
        else
```

Reply:
Seems this change only affect the checksum ,not the binders themselves.  Do you mean we should change something in mbedtls_ssl_tls13_write_pre_shared_key_ext_binders() or it's parameters?

Reply:
Done.